app/src/main/res/layout/toolbar_navigation.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this an unrelated cleanup? I'd appreciate to put this into a separate PR, so its easier to roll back in case this actually still used somewhere.